web-ui/src/main/resources/catalog/templates/editor/editor.html

```diff
@@ -1,7 +1,10 @@
 <div data-ng-hide="metadataFound">
-  <div class="alert alert-warning" 
+  <div data-ng-show="!metadataLocked" class="alert alert-warning" 
     data-translate="" 
     data-translate-values="{ id: '{{metadataNotFoundId}}'}">metadataNotFound</div>
+    
+  <div data-ng-show="metadataLocked" class="alert alert-warning" 
```

Comment:
> It has to be because Github is slow at updating the code. I pushed a change that allows it: 5687ad0

Would be good to have a way to go somewhere in this case. Only the browser back button allows to do something

![image](https://cloud.githubusercontent.com/assets/1701393/15422538/6224b5c6-1e79-11e6-8673-b25df54401b6.png)


Also when requesting a non existing record the metadata lock message is displayed

![image](https://cloud.githubusercontent.com/assets/1701393/15422586/9a4cd47e-1e79-11e6-8b9b-db33f40d4db0.png)

which is not correct.